src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -795,7 +795,6 @@ public static SyntaxKind GetTypeDeclarationKind(SyntaxKind kind)
                 case SyntaxKind.InterfaceKeyword:
                     return SyntaxKind.InterfaceDeclaration;
                 case SyntaxKind.RecordKeyword:
-                    // PROTOTYPE(record-structs): anything we can do? 
```

Comment:
:memo: I don't see anything we could do given the shape of the API. We'll see if that causes problems during IDE work